### PR TITLE
fix: make sure we are not using the deprecated resolved_capabilites

### PR DESCRIPTION
**Pull request by abzcoding**
use `server_capabilites` instead of the deprecated `resolved_capabilites`

more info:
- https://www.reddit.com/r/neovim/comments/u5si2w/breaking_changes_inbound_next_few_weeks_for/
- https://github.com/neovim/neovim/commit/c618b314c6a266806edf692122b16ba9ff7a8e10

### CODE REVIEW

File: lua/lsp_signature/helper.lua [hunk 1]
```diff
@@ -514,17 +514,24 @@ function helper.check_lsp_cap(clients, line_to_cursor)
...
   local trigger_position = nil
+  local vim_version = vim.version()
 
   local tbl_combine = helper.tbl_combine
   for _, value in pairs(clients) do
     if value ~= nil then
       local sig_provider = value.server_capabilities.signatureHelpProvider
-      local rslv_cap = value.resolved_capabilities
+      local rslv_cap = value.server_capabilities
+      if vim_version.minor < 8 then
```
Comment (ray-x) [inline lua/lsp_signature/helper.lua:524]: Maybe use `if vim_version.minor < 6 then`
I run a quick test against nvim 0.6.1 and it works fine. In fact, I am not planing to support versions lower than 0.6.1 so you can 
1. check version, if it is lower than 0.6.1(minor 6, patch 1) send a warn/notify and return
2. if the version is higher than 0.6.1 (minor >=7) use server_capabilities

Thanks!

**Review by ray-x — COMMENTED** [review]
No support for versions lower than 0.6. Thanks!

File: lua/lsp_signature/helper.lua (hunk 1 above)
Comment (abzcoding) [reply lua/lsp_signature/helper.lua]: i do like the first option :)

**Review by ray-x — APPROVED** [review]
Thanks!

**Pull request merged**